stargate/crates/query-planner/tests/features/basic/csdl.graphql

```diff
@@ -5,7 +5,7 @@ schema
 @graph(name: "inventory", url: "https://inventory.api.com")
 @graph(name: "product", url: "https://product.api.com")
 @graph(name: "reviews", url: "https://reviews.api.com")
-@composedGraph(version: 1)
+@using(spec: "http://specs.apollo.dev/cs/v0.1")
```

Comment:
One thought is should we maybe commit to the URL before finalizing this?

Reply:
Worth noting that this _has_ to be `https://` because `.dev` is Strict Transport Security (HSTS) and requires `https://`.